pyproject.toml

```diff
@@ -35,7 +35,7 @@ dependencies = [
         'django-localflavor==4.0',
         'reportlab==4.0.*',
         'Pillow==9.5.*',
-        'PyPDF2==2.12.*',
+        'PyPDF2==3.0.*',
```

Comment:
Why don't you update to version 4.2? Any errors with a higher version?

Reply:
Hi @mariobehling, latest version for 'PyPDF2' is 3 (https://pypi.org/project/PyPDF2/)
4.2 version you mention it may from different package https://pypi.org/project/pypdf/

Reply:
@odkhang Pretix uses pypdf==4.2.* instead of PyPDF2.

Reply:
`pypdf` version 4 is the successor of `PyPDF2` version 3. They moved back to the original naming.

Reply:
Hi @mariobehling, @norbusan thanks a lot for the info, I have updated it accordingly, please help me to review it.